src/app/components/movie.card/movie.card.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import './movie.card.css'
+import MovieCover from "./movie.cover";
+import MovieTitle from "./movie.title";
+import MovieDescription from "./movie.description";
+import MovieYear from "./movie.year";
+
+
+export default function MovieCard(props) {
+    return (
+        <div className='MovieCard'>
+            <MovieCover cover={props.cover}/>
+            <div className="under-cover">
+                <MovieTitle title={props.title}/>
+                <MovieYear year={props.year}/>
+            </div>
+            <MovieDescription description={props.description}/>
+        </div>
+    )
+}
+
+MovieCard.propTypes = {
```

Comment:
You don't need propTypes since you are using typescript. You can add a type for MovieCard props like `type MovieCardProps = {
  title: string,
    description: string,
    year?: number,
    cover?: string,
};`

Reply:
Even though it may seem like Typescript and PropTypes do the same thing, they don't actually overlap at all. But it is possible to automatically generate PropTypes from Typescript so that you don't have to specify types twice, see for example:

https://github.com/milesj/babel-plugin-typescript-to-proptypes
https://github.com/grncdr/ts-react-loader#what-it-does
https://github.com/gcanti/prop-types-ts